internal/common/shared/gifts.go

```diff
@@ -1,7 +1,28 @@
 package shared
 
-// GiftDict is a dictonary of gift
-type GiftDict = map[ClientID]uint
+// GiftRequest contains the details of a gift request from an island to another
```

Comment:
Me being a dumb dumb in English got confused with the naming scheme. I had to ask for clarification from the rest of Team1. Is it okay to be more explicit in the comments (same for the below)? 

Moreover, OfferingTeam is used again in GiftOffer - does it still hold the same meaning?

Reply:
as @mpardalos pointed out, the 2 data types are pretty similar for now, they may not have to be separate. They just are for the time being until I make test cases to execute the code, and we know that they don't have to be different for data protection purposes.

Reply:
Okay. I will wait for this to be completed. Resolving convo for now.